webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <base-card class="follow-list">
+    <template v-if="connections && connections.length">
+      <h5 class="title spacer-x-small">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </h5>
+      <div :class="connectionsClass">
+        <user-teaser
+          v-for="connection in filteredConnections"
+          :key="connection.id"
+          :user="connection"
+          class="spacer-x-small"
+        />
+      </div>
+      <base-button
+        v-if="allConnectionsCount - connections.length"
+        :loading="loading"
+        class="spacer-x-small"
+        size="small"
+        @click="$emit('fetchAllConnections', type)"
+      >
+        {{
+          $t('profile.network.andMore', {
+            number: allConnectionsCount - connections.length,
+          })
+        }}
+      </base-button>
+      <ds-input
+        v-if="connections.length > 7"
+        v-focus="true"
+        :name="`${type}Filter`"
+        :placeholder="filter"
+        class="spacer-x-small"
+        icon="filter"
+        size="small"
+        @input.native="setFilter"
+      />
+    </template>
+    <p v-else class="nobody-message">{{ userName }} {{ $t(`profile.network.${type}Nobody`) }}</p>
+  </base-card>
+</template>
+
+<script>
+import { escape } from 'xregexp/xregexp-all.js'
+import UserTeaser from '~/components/UserTeaser/UserTeaser'
+
+export default {
+  name: 'FollowerList',
+  components: {
+    UserTeaser,
+  },
+  props: {
+    user: { type: Object, default: null },
+    type: { type: String, default: 'following' },
+    loading: { type: Boolean, default: false },
+  },
+  data() {
+    return {
+      filter: null,
+    }
+  },
+  computed: {
+    userName() {
+      const { name } = this.user || {}
+      return name || this.$t('profile.userAnonym')
+    },
+    allConnectionsCount() {
+      return this.user[`${this.type}Count`]
+    },
+    connections() {
+      return this.user[this.type]
+    },
+    connectionsClass() {
+      const overflow = this.connections.length > 7 ? ' --overflow' : ''
+      return `connections${overflow}`
```

Comment:
Since it's just one property you could maybe just
```suggestion
      return `connections${this.connections.length > 7 ? ' --overflow' : ''}`
```

Reply:
> The profile page had the count of displayed connections hardcoded to 7; I just took that over, when I started.
It makes way more sense now to introduce a `hasMore` computed value of `user.followingCount > user.following.length`

Reply:
e018e2ead